code/controllers/subsystem/discord.dm

```diff
@@ -0,0 +1,110 @@
+/* 
+NOTES:
+There is a DB table to track ckeys and associated discord IDs.
+This system REQUIRES TGS, and will auto-disable if TGS is not present.
+The SS uses fire() instead of just pure shutdown, so people can be notified if it comes back after a crash, where the SS wasnt properly shutdown
+It only writes to the disk every 5 minutes, and it wont write to disk if the file is the same as it was the last time it was written. This is to save on disk writes
+The system is kept per-server (EG: Terry will not notify people who pressed notify on Sybil), but the accounts are between servers so you dont have to relink on each server.
+
+##################
+# HOW THIS WORKS #
+##################
+
+ROUNDSTART:
+1] The file is loaded and the discord IDs are extracted
+2] A ping is sent to the discord with the IDs of people who wished to be notified
+3] The file is emptied
+
+MIDROUND: 
+1] Someone usees the notify verb, it adds their discord ID to the list.
+2] On fire, it will write that to the disk, as long as conditions above are correct
+
+END ROUND:
+1] The file is force-saved, incase it hasnt fired at end round
+
+This is an absolute clusterfuck, but its my clusterfuck -aa07
+*/
+
+SUBSYSTEM_DEF(discord)
+	name = "DIscord"
```

Comment:
Yes I know this is badly capitalised, I will fix this when I have to make the fix PRs for the inevitable other issues in this